taskcluster/docker/android-build/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# %ARG DOCKER_IMAGE_PARENT
```

Comment:
nit: Do we add license headers to those files?

Reply:
Oops, done!